src/functions/vacation-request/update-vacation-request/handler.ts

```diff
@@ -0,0 +1,71 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from "@libs/api-gateway";
+import { middyfy } from "@libs/lambda";
+import { vacationRequestService } from "src/database/services";
+import type VacationRequestModel from "src/database/models/vacationRequest";
+import type vacationRequestSchema from "src/schema/vacationRequest";
+
+/**
+ * Lambda function to update a vacation request
+ *
+ * @param event event
+ */
+const updateVacationRequestHandler: ValidatedEventAPIGatewayProxyEvent<typeof vacationRequestSchema> = async event => {
+  const { pathParameters, body } = event;
+  const id = pathParameters?.id;
+  const {
+    personId,
+    draft,
+    startDate,
+    endDate,
+    days,
+    type,
+    message,
+    createdBy,
+    createdAt,
+    updatedAt
+  } = body;
```

Comment:
Do we need some kind of validation check for these e.g. similar to the create function where is a required attribute is missing we return a 400 "invalid body" response?